### DISCUSSION TITLE
Strange value in "Set Sigen ESS max (dis)charging limit"

### DISCUSSION THREAD

**Discussion by Wictorssons** [TypQxQ/Sigenergy-Home-Assistant-Integration#9] [General]
I noticed that the "Set Sigen ESS max charging limit" (and discharging) fields both have the value 4294967.295.
Any idea where these come from?

![image](https://github.com/user-attachments/assets/1a454960-c275-4329-b13e-74cf8cd8db1a)

**Comment by TypQxQ** [comment]
It is the value when it has never been initialized. The max uint32 value. /Andrei17 sep. 2024 kl. 22:24 skrev Wictorsson ***@***.***>:﻿
I noticed that the "Set Sigen ESS max charging limit" (and discharging) fields both have the value 4294967.295.
Any idea where these come from?
image.png (view on web)
